cmd/axiom/auth/auth_login.go

```diff
@@ -146,63 +171,29 @@ func completeLogin(opts *loginOptions) error {
 		hostRef = ""
 	}
 
-	questions := make([]*survey.Question, 0, 3)
-
+	// 5. Ask for an alias to use.
 	if opts.Alias == "" {
-		questions = append(questions, &survey.Question{
-			Name: "alias",
-			Prompt: &survey.Input{
-				Message: "Under which name should the deployment be referenced in the future?",
-				Default: hostRef,
-			},
-			Validate: survey.ComposeValidators(
-				survey.Required,
-				survey.MinLength(5),
-			),
-		})
-	}
-
-	if opts.OrganizationID == "" && deploymentKind == typeCloud {
-		questions = append(questions, &survey.Question{
-			Name:   "organizationID",
-			Prompt: &survey.Input{Message: "What is your organization ID?"},
-			Validate: survey.ComposeValidators(
-				survey.Required,
-				survey.MinLength(12),
-				survey.MaxLength(96),
-			),
-		})
-	}
-
-	if opts.TokenType == "" {
-		questions = append(questions, &survey.Question{
-			Name: "tokenType",
-			Prompt: &survey.Select{
-				Message: "What kind of token will you provide?",
-				Options: validTokenTypes,
-			},
-		})
-	}
-
-	if opts.Token == "" {
-		questions = append(questions, &survey.Question{
-			Name:   "token",
-			Prompt: &survey.Password{Message: "What is your personal access or ingest token?"},
-			Validate: survey.ComposeValidators(
-				survey.Required,
-				survey.MinLength(36),
-				survey.MaxLength(36),
-			),
-		})
+		if err := survey.AskOne(&survey.Input{
+			Message: "Under which name should the deployment be referenced in the future?",
+			Default: hostRef,
+		}, &opts.Alias, survey.WithValidator(survey.ComposeValidators(
+			survey.Required,
+			survey.MinLength(3),
+		)), opts.IO.SurveyIO()); err != nil {
+			return err
+		}
 	}
 
-	return survey.Ask(questions, opts, opts.IO.SurveyIO())
+	return nil
 }
 
 func runLogin(ctx context.Context, opts *loginOptions) error {
-	// Read token from stdin, if the appropriate option is set.
-	if opts.TokenStdIn {
-		contents, err := ioutil.ReadAll(opts.IO.In())
+	// Read token from stdin, if no TTY is attached.
+	if !opts.IO.IsStdinTTY() {
+		// The token won't be longer.
+		r := io.LimitReader(opts.IO.In(), 64)
```

Comment:
We can't know that tokens will always be shorter. Does it make sense use something like 256 so we have some room?